src/main/java/qna/domain/Answer.java

```diff
@@ -17,17 +21,20 @@
 import java.util.Objects;
 
 @Entity
-@Table(name = "answer")
 public class Answer extends BaseDateEntity{
 
     @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @Lob
     private String contents;
-    private Long questionId;
     @Column(nullable = false)
     private boolean deleted = false;
-    private Long writerId;
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "QUESTION_ID")
+    private Question question;
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "WRITER_ID")
+    private User user;
```

Comment:
```suggestion
    private User writer;
```
`@JoinColumn`을 생략하고 변수명을 writer로 쓰면 어떻게 동작할까요?

Reply:
현재 양방향참조관계를 설정하여, @JoinColumn을 생략하고 변수명을 writer로만 바꾸면 빌드 시 에러가 발생하게 됩니다. 연관관계 매핑 중 mappedBy 값 또한 writer로 변경해줘야지 원하는대로 동작하게 됩니다.

해당 부분은 리뷰어님이 지적 해주시기 전까지 크게 고민해보지 않은 사항인데, 리뷰어님 덕분에 한번 수행해보고 내부적으로 어떻게 처리되는지 확인하게 되었습니다.
감사합니다 👍